packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -298,6 +302,35 @@ function useMediaUploadQueue() {
     [startMuting, finishMuting, stripAudioFromVideo, cancelUploading]
   );
 
+  const cropVideoItem = useCallback(
+    async (item) => {
+      const { id, file, additionalData } = item;
+
+      startCropping({ id });
+
+      currentTranscodingItem.current = id;
+
+      try {
+        const newFile = await cropResource(file, additionalData.cropParams);
+
+        if (!isMounted.current) {
+          return;
+        }
+
+        additionalData.isCropped = true;
+        finishCropping({ id, file: newFile, additionalData });
```

Comment:
```suggestion
         const newFile = await cropResource(file, additionalData.cropParams);
         const posterFile = await getFirstFrameOfVideo(newFile);

        if (!isMounted.current) {
          return;
        }

        additionalData.isCropped = true;
        finishCropping({ id, file: newFile, posterFile, additionalData });
```

The following + updating to finishCropping to accept posterFile worked well for me.